Test failure: JIT/opt/SVE/PredicateInstructions/PredicateInstructions.cmd

**Failed in:** [runtime-coreclr jitstress-isas-arm 20250524.1](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1050425&view=ms.vss-test-web.build-test-results-tab&runId=28301748&resultId=119466&paneView=debug)

**Failed tests:**
```
coreclr windows arm64 Checked jitstress_isas_nohwintrinsic @ Windows.11.Arm64.Open
    - JIT/opt/SVE/PredicateInstructions/PredicateInstructions.cmd
```

**Error message:**
```
 __tmp0_PredicateInstructions.cs:35:11: error: ARM64: expected string not found in input
 //ARM64: {{^ *}} zip1 {{p[0-9]+}}.h, {{p[0-9]+}}.h, {{p[0-9]+}}.h{{$}}
          ^
__jit_disasm.out:19:46: note: scanning from here
; BEGIN METHOD PredicateInstructions:ZipLow():System.Numerics.Vector`1[short]
                                             ^
__jit_disasm.out:22:2: note: possible intended match here
 stp fp, lr, [sp, #-0x10]!
 ^

Input file: __jit_disasm.out
Check file: __tmp0_PredicateInstructions.cs

-dump-input=help explains the following input dump.

Input was:
<<<<<<
            1: ; Assembly listing for method PredicateInstructions:ZipLow():System.Numerics.Vector`1[short] (FullOpts) 
            2: ; Emitting BLENDED_CODE for generic ARM64 - Windows 
            3: ; FullOpts code 
            4: ; optimized code 
            5: ; fp based frame 
            6: ; partially interruptible 
            7: ; No PGO data 
            8: ; 0 inlinees with PGO data; 3 single block inlinees; 1 inlinees without PGO data 
            9: ; Final local variable assignments 
           10: ; 
           11: ;# V00 OutArgs [V00 ] ( 1, 1 ) struct ( 0) [sp+0x00] do-not-enreg[XS] addr-exposed "OutgoingArgSpace" <Empty> 
           12: ;* V01 tmp1 [V01 ] ( 0, 0 ) simd16 -> zero-ref "impAppendStmt" 
           13: ;* V02 tmp2 [V02 ] ( 0, 0 ) simd16 -> zero-ref "spilled call-like call argument" 
           14: ;* V03 tmp3 [V03 ] ( 0, 0 ) simd16 -> zero-ref ld-addr-op "Inline ldloca(s) first use temp" <System.Numerics.Vector`1[short]> 
           15: ;* V04 tmp4 [V04 ] ( 0, 0 ) simd16 -> zero-ref "dummy temp of must thrown exception" <System.Numerics.Vector`1[short]> 
           16: ;* V05 tmp5 [V05 ] ( 0, 0 ) simd16 -> zero-ref "dummy temp of must thrown exception" <System.Numerics.Vector`1[short]> 
           17: ; 
           18: ; Lcl frame size = 0 
           19: ; BEGIN METHOD PredicateInstructions:ZipLow():System.Numerics.Vector`1[short] 
check:35'0                                                  X~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ error: no match found
           20:  
check:35'0     ~
           21: G_M53919_IG01: 
check:35'0     ~~~~~~~~~~~~~~~
           22:  stp fp, lr, [sp, #-0x10]! 
check:35'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~
check:35'1      ?                          possible intended match
           23:  mov fp, sp 
check:35'0     ~~~~~~~~~~~~
           24:  ;; size=8 bbWeight=1 PerfScore 1.50 
check:35'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           25: G_M53919_IG02: 
check:35'0     ~~~~~~~~~~~~~~~
           26:  bl CORINFO_HELP_THROW_PLATFORM_NOT_SUPPORTED 
check:35'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           27:  brk #0 
check:35'0     ~~~~~~~~
           28:  ;; size=8 bbWeight=1 PerfScore 2.00 
check:35'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           29: ; END METHOD PredicateInstructions:ZipLow():System.Numerics.Vector`1[short] 
check:35'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           30:  
check:35'0     ~
           31: ; Total bytes of code 16, prolog size 8, PerfScore 3.50, instruction count 4, allocated bytes for code 16 (MethodHash=14112d60) for method PredicateInstructions:ZipLow():System.Numerics.Vector`1[short] (FullOpts) 
check:35'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           32: ; ============================================================ 
check:35'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           33:  
check:35'0     ~
>>>>>>
__tmp1_PredicateInstructions.cs:37:17: error: CHECK-LABEL: expected string not found in input
// CHECK-LABEL: BEGIN METHOD PredicateInstructions:ZipHigh({{.*}})
                ^
__jit_disasm.out:1:1: note: scanning from here
; Assembly listing for method PredicateInstructions:ZipLow():Syst
```

**Stack trace:**
```
   at Xunit.Assert.True(Nullable`1 condition, String userMessage) in /_/src/arcade/src/Microsoft.DotNet.XUnitAssert/src/BooleanAsserts.cs:line 141
   at Program.<<Main>$>g__TestExecutor372|0_373(StreamWriter tempLogSw, StreamWriter statsCsvSw, <>c__DisplayClass0_0&)
```

--- a/src/tests/JIT/opt/SVE/PredicateInstructions.cs
+++ b/src/tests/JIT/opt/SVE/PredicateInstructions.cs
@@ -15,18 +15,21 @@ public class PredicateInstructions
     [Fact]
     public static void TestPredicateInstructions()
     {
-        ZipLow();
-        ZipHigh();
-        UnzipOdd();
-        UnzipEven();
-        TransposeOdd();
-        TransposeEven();
-        ReverseElement();
-        And();
-        BitwiseClear();
-        Xor();
-        Or();
-        ConditionalSelect();
+        if (Sve.IsSupported)
+        {
+            ZipLow();
+            ZipHigh();
+            UnzipOdd();
+            UnzipEven();
+            TransposeOdd();
+            TransposeEven();
+            ReverseElement();
+            And();
+            BitwiseClear();
+            Xor();
+            Or();
+            ConditionalSelect();
+        }
     }
 
     [MethodImpl(MethodImplOptions.NoInlining)]

